templates/search/_filters.html.twig

```diff
@@ -4,11 +4,20 @@
 
         {% if search.isTypeEvents() %}
             <div class="search__bar__options__types">
-                <select id="select-event-category" name="{{ constant('PARAMETER_EVENT_CATEGORY', search) }}" class="form">
-                    <option value=""{% if not search.eventCategory %} selected="selected"{% endif %}>Tous les événements</option>
+                <select id="select-event-category" name="{{ constant('PARAMETER_EVENT_CATEGORY', search) }}"
+                        class="form">
+                    <option value=""{% if not search.eventCategory %} selected="selected"{% endif %}>Tous les
+                        événements
+                    </option>
                     <option value="{{ constant('TYPE_CITIZEN_ACTIONS', search) }}">Actions citoyennes</option>
                     {% for category in event_categories %}
-                        <option value="{{ category.id }}"{% if search.eventCategory == category.id %} selected="selected"{% endif %}>{{ category.name }}</option>
+                        <optgroup label="{{ category.name }}">
+                            {% for category in category.eventCategories %}
+                                {% if category.status == 'ENABLED' %}
```

Comment:
IMO it's not a good place for this check

Reply:
removed